core/src/main/java/com/graphhopper/routing/weighting/custom/CustomWeighting.java

```diff
@@ -169,4 +165,46 @@ public long calcEdgeMillis(EdgeIteratorState edgeState, boolean reverse) {
     public String getName() {
         return NAME;
     }
+
+    @FunctionalInterface
+    public interface EdgeToDoubleMapping {
+        double apply(EdgeIteratorState edge, boolean reverse);
```

Comment:
In current master this is named `EdgeToValueEntry` and `getValue` but the class name you have is better, unsure about apply :D

Reply:
`java.util.Function` calls it `apply`, but honestly this never sounded very intuitive to me either. Maybe call it `get` like `java.util.Supplier`? Or `map`? Don't know really.

Reply:
`get` sounds better, yes :)